testdata/regresstest/config

```diff
@@ -452,6 +452,13 @@ check_checkpoint_eq adpf.gap unused shape5pfx "CGAGCGUAGUCGUAGCguCGUAGCGUAGUCGUA
 GAPC="../../../gapc --checkpoint --sample"
 # double + Backtrace
 check_checkpoint_eq adpf.gap unused pfsampleshape "CGAGCGGUACUGAGUCGUAGUGUAGUGCUCGUAGUCGUAGUCGUAGUCGUAGCUGAUCGUAGUAGUCGUCGUAGUCGUAGUCGUCGUAGCGUAGUCGUAGUCGUAGUCAGUCGUGUCGUGUACGUAGUCGUCAGUGCGAUCGUGUCGUAGUCGUCGUAGUCGUAGUCGAGUCAGUCGUAGUAGUGCUGAUCGUgauCGUAGCGUACGUACGUAGUCGUAGUCGUAGUCGUAGUCGAGUCCGUAGUCGUAGUCGUAGUCGUAGUCGUGUACGUACGUGUACGGUCGUAGUCGUAGCUguCGUAGCGUAGUCGUAGUCGUAGUCGUAGUCAGUCGUAGUGUAGUGUCGUCGUCGUAGUCGUAGUCGUAGUCAGUCGUAGUCGUAGUCGUAGUCGAUGUAGUGUCGUAGUCAGUCUGAGUCGUAGCGUAGUCGAUGUCAGUCGUAGUCGUAGUCGUAGUCGAUGCUAGUCCUGAGUCGUAGUCGUAGUCGAUCGUAGCUGAUCGUCGUAGUCGUAGUCGAGUCGACGUAGUCGUACGUAGUCGUACGUAGUCGUAGUACGUAGUGUACguCGUA" checkpoint7 1
+GAPC="../../../gapc --checkpoint --cyk"
+# std::pair<int, double> + cyk
+check_checkpoint_eq adpf.gap unised mfepf "CGUAGCUGAUGCUGAGCGUCGUAUCUGAGUCGUGUAGUCGAUGUCGAUCGUAGUCGUGUCGAGUCGUAUGCGUAGCGUAGUCGUAGCUAGUCCGUAGCGUAGUCAGUGCUAGUGCUGCAGUGCUGAUCGUGUACGUCGAGUCGAGUCGUAGUCGUACGUAGUGCUAGUGCUAGUCGUAGUCGUGUCGUAGUCGUGAUGUCAGUGUCGUGUAGUCGUAGUCGUACGUGUACGUAGUCGUAGUCGUAUCGAGUCGCGUAGGCUAGUCGUAGUCGUAGUCGUGCGUAGCGUUGAGUCGUAGUCGAUGUCAGUCGUGUAGUCUGAGCUGAUCUGUUCGUAGCGUGUACGAGUCGUGUCGUGUAGCUGUAGUCGUAGUCAGUCAUGUAGUCGAUGCGUAGUCGUGUACGUGUAGUCGUAGUCGUAGUCGUGUAGUCGUAGUCGUAGUGUCGUAGCUGUAGUCGUGUAGUCGUAGUCUGAGUGUCGUAGUCGUCGUGUAGUCGUGUACGUGUAGUCGUGUAGUCGUAGUCGUAGUCGUUAUCGGAUGUCGUAGCUGUACGUGAUCGUCCGUC" checkpoint8 1
+# std::pair<int, Rope> + cyk
+check_checkpoint_eq adpf.gap unused mfeen "CGAGCGGUACUGAGUCGUAGUGUAGUGCUCGUAGUCGUAGUCGUAGUCGUAGCUGAUCGUAGUAGUCGUCGUAGUCGUAGUCGUCGUAGCGUAGUCGUAGUCGUAGUCAGUCGUGUCGUGUACGUAGUCGUCAGUGCGAUCGUGUCGUAGUCGUCGUAGUCGUAGUCGAGUCAGUCGUAGUAGUGCUGAUCGUgauCGUAGCGUA" checkpoint9 1
+# std::pair<int, String> + cyk
+check_checkpoint_eq adpf.gap unused mfepp "CGAGCGGUACUGAGUCGUAGUGUAGUGCUCGUAGUCGUAGUCGUAGUCGUAGCUGAUCGUAGUAGUCGUCGUAGUCGUAGUCGUCGUAGCGUAGUCGUAGUCGUAGUCAGUCGUGUCGUGUACGUAGUCGUCAGUGCGAUCGUGUCGUAGUCGUCGUAGUCGUAGUCGAGUCAGUCGUAGUAGUGCUGAUCGUgauCGUAGCGUAGUCGAUCGUGAUGUCGUAGUCGUGUAGCUGAUGCUGUAGUCGUAUGCGUAGUCGUGUACGUGUAGCU" checkpoint10 1
 GAPC="../../../gapc"
```

Comment:
can you please also test multi track grammars like `grammar/ali2.gap`?

Reply:
Will do.